components/eam/src/physics/cam/shoc_intr.F90

```diff
@@ -438,6 +447,24 @@ subroutine shoc_init_e3sm(pbuf2d, dp1_in)
     call add_default('LIQ_CLOUD_FRAC',1,' ')
     call add_default('TOT_CLOUD_FRAC',1,' ')
 
+    ! Add output variables from SHOC's internal substeps
+
+    if (turb_nadv_out_nstep.gt.0) then
+       do iadv = 1,turb_nadv_out_nstep
+
+          write(numstr,'(a1,i4.4)') '_',iadv
+
+          call addfld('SHOC_TKE'//trim(adjustl(numstr)), (/'lev'/), 'A', 'm2/s2', 'TKE')
+          call addfld(  'THETAL'//trim(adjustl(numstr)), (/'lev'/), 'A', 'K',     'liquid water potential temperature')
+          call addfld( 'CLDFRAC'//trim(adjustl(numstr)), (/'lev'/), 'A', '1',     'cloud fraction')
```

Comment:
Same comment as above regarding 'A' and 'I'.

Also, should we use units "fraction" instead of "1" for consistency with the other fraction-related variables?